### ISSUE TITLE
Deprecate `babel-code-frame` on NPM

### ISSUE THREAD

**Issue by jonkoops** [babel/babel#16994]
### 💻

- [ ] Would you like to work on this feature?

### What problem are you trying to solve?

The legacy `babel-code-frame` package is still downloaded over 5 million times. It should probably be marked as [deprecated on NPM](https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions).

### Describe the solution you'd like

For `babel-code-frame` to be deprecated on NPM so users are guided to install the actively supported `@babel/code-frame` package.

### Describe alternatives you've considered

Keep things as is and leave users unaware.

**Comment by jonkoops** [comment]
@hzoo FYI

**Comment by nicolo-ribaudo** [comment]
Yeah, we should probably do this with all the Babel 6 packages. @JLHwung @liuxingbaoyu what do you think?
